Makefile

```diff
@@ -1,15 +1,37 @@
+# Environment Variables
 DEPLOYMENT_ENVIRONMENT?=localstack
+LOCALSTACK_AUTH_TOKEN?=set-localstack-auth-token
 RUNNER_IMAGE_NAME?=dev-image
 DOCKER_SOCK_MOUNT?=-v /var/run/docker.sock:/var/run/docker.sock
-DOCKER_BUILD_EXTRA_ARGS?=--build-arg="TERRAFORM_VERSION=1.7.3" --build-arg="NODE_MAJOR=20" --build-arg="CHECKOV_VERSION=3.1.40"
+DOCKER_BUILD_EXTRA_ARGS?=--build-arg="TERRAFORM_VERSION=1.7.3" \
+                         --build-arg="NODE_MAJOR=20" \
+                         --build-arg="CHECKOV_VERSION=3.1.40"
 DOCKER_RUN_MOUNT_OPTIONS:=-v ${PWD}:/app -w /app
 AWS_DEFAULT_REGION?=ap-south-1
 AWS_REGION?=$(AWS_DEFAULT_REGION)
+
+# Docker Environment Variables
 TF_VARS=$(shell env | grep '^TF_VAR_' | awk '{print "-e", $$1}')
-DOCKER_ENV?=-e AWS_ACCESS_KEY_ID -e DEPLOYMENT_ENVIRONMENT -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e AWS_DEFAULT_REGION -e AWS_REGION -e TF_BACKEND_BUCKET_NAME -e TF_BACKEND_BUCKET_REGION -e TF_BACKEND_BUCKET_KEY $(TF_VARS)
-TF_BACKEND_CONFIG=--backend-config="bucket=$(TF_BACKEND_BUCKET_NAME)" --backend-config="key=$(TF_BACKEND_BUCKET_KEY)" --backend-config="region=$(TF_BACKEND_BUCKET_REGION)"
+DOCKER_ENV?=-e AWS_ACCESS_KEY_ID \
+            -e DEPLOYMENT_ENVIRONMENT \
+            -e AWS_SECRET_ACCESS_KEY \
+            -e AWS_SESSION_TOKEN \
+            -e AWS_DEFAULT_REGION \
+            -e AWS_REGION \
+            -e TF_BACKEND_BUCKET_NAME \
+            -e TF_BACKEND_BUCKET_REGION \
+            -e TF_BACKEND_BUCKET_KEY $(TF_VARS)
+
+# Terraform Backend Configuration
+TF_BACKEND_CONFIG=--backend-config="bucket=$(TF_BACKEND_BUCKET_NAME)" \
+                  --backend-config="key=$(TF_BACKEND_BUCKET_KEY)" \
+                  --backend-config="region=$(TF_BACKEND_BUCKET_REGION)"
+
+# Checkov Skip Rules
```

Comment:
please add why we are skipping these rurles

Reply:
updated in b6b119a6dd73086ee41929ddbaedf2483cca9b9c